LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 UGRC
+Copyright (c) 2024 UGRC
```

Comment:
i might suggest removing the year to not have to think about it.

Reply:
Github adds the year by default if you choose the mit template which is probably why it was initially there. I've been removing it everywhere I notice it since I learned that the year was not required. This sums it up pretty well. https://opensource.stackexchange.com/a/5779. I'm open to whatever but my preference is to remove it.